provider/lpwindow/compute_task.go

```diff
@@ -161,6 +161,14 @@ func (t *WdPostTask) Do(taskID harmonytask.TaskID, stillOwned func() bool) (done
 		return false, xerrors.Errorf("marshaling PoSt: %w", err)
 	}
 
+	testTaskID := 0
+	testTaskIDCt := 0
+	if err = t.db.QueryRow(context.Background(), `SELECT COUNT(*) FROM harmony_test WHERE task_id = $1`, taskID).Scan(&testTaskIDCt); err != nil {
+		return false, xerrors.Errorf("querying for test task: %w", err)
+	}
+	if testTaskIDCt == 1 {
+		testTaskID = int(taskID)
+	}
 	// Insert into wdpost_proofs table
 	n, err := t.db.Exec(context.Background(),
```

Comment:
We want to insert into the harmony_test table here, to avoid passing through the submit task, which won't run until the deadline open time - which can be up to a day of waiting.

It a PoSt test run SPs just want to get the compute result as soon as it's computed, and see the proof / skipped sector number. Submit task doesn't add much other than waiting for the correct time + wrapping the PoSt into a chain message, but neither of those useful to SP operators

Reply:
The Sender works for Do(), so we avoid the submit task. It also gets the nonce, which I figured may be a nice addition to show how completed the (likely inaccurate) windowpost test got.

harmony_test is both input and output to this task: 
 - input if this task is a test, in Do()
 - output the test result (below) and consider complete, in Send()